chromium_src/chrome/browser/ui/tabs/split_tab_menu_model.h

```diff
@@ -23,8 +23,11 @@
   }                                 \
   virtual const gfx::VectorIcon& GetReversePositionIcon(__VA_ARGS__)
 
+#define kExitSplit kExitSplit, kToggleLinkState
```

Comment:
Could you add comments for this `#deinfe` saying that we're adding new CommandId

Reply:
Added.